modules/juce_audio_devices/native/juce_ios_Audio.cpp

```diff
@@ -982,6 +1027,12 @@ struct iOSAudioIODevice::Pimpl      : public AudioPlayHead,
                 
         setAnalogInputGain(0.5f);
         
+        if (!AudioIODeviceType::useDeviceVoiceProcessing) {

```

Comment:
Do we also care about isUsingBuiltInSpeaker() in the context of mic selection?

Reply:
True, I'll add the condition